CONTRIBUTING.md

```diff
@@ -20,13 +20,27 @@ Before creating an issue, please check the following:
 > **Warning**
 > Do not close issues that are about to be resolved. It should remain open until a commit that actually resolves it is merged.
 
-## Before implementation
+
+### Recommended discussing before implementation
+We welcome your purposal.
+
 When you want to add a feature or fix a bug, **first have the design and policy reviewed in an Issue** (if it is not there, please make one). Without this step, there is a high possibility that the PR will not be merged even if it is implemented.
 
 At this point, you also need to clarify the goals of the PR you will create, and make sure that the other members of the team are aware of them.
 PRs that do not have a clear set of do's and don'ts tend to be bloated and difficult to review.
 
-Also, when you start implementation, assign yourself to the Issue (if you cannot do it yourself, ask another member to assign you). By expressing your intention to work the Issue, you can prevent conflicts in the work.
+Also, when you start implementation, assign yourself to the Issue (if you cannot do it yourself, ask Commiter to assign you). By expressing your intention to work the Issue, you can prevent conflicts in the work.
+
+To the Commiters: you should not assign someone on it before the Final Desicion.
```

Comment:
Commiters is not the correct spelling it is Committers with 2 T's
Same applies to `Commiter`

Desicion is also not the correct spelling it is Decision simply typo caused by swapping characters

Reply:
`By expressing your intention to work on the Issue`